plugins/factory-plugin/src/workspace-projects-manager.ts

```diff
@@ -195,7 +191,7 @@ export class WorkspaceConfigProjectsManager extends WorkspaceProjectsManager {
 
         projectsHelper.updateOrCreateGitProjectInWorkspaceConfig(
             workspace.config.projects,
-            fileUri.convertToCheProjectPath(projectFolderURI, this.projectsRoot),
+            '/' + fileUri.convertToCheProjectPath(projectFolderURI, this.projectsRoot),
```

Comment:
Maybe  `path.sep` here instead of '/'?  But up to you.

Reply:
The same as above